internal/docker/env.go

```diff
@@ -5,13 +5,17 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/blang/semver"
 	"github.com/docker/cli/opts"
 	"github.com/pkg/errors"
 
 	"github.com/windmilleng/tilt/internal/container"
 	"github.com/windmilleng/tilt/internal/k8s"
+	"github.com/windmilleng/tilt/pkg/logger"
 )
 
+var minMinikubeVersionBuildkit = semver.MustParse("1.8.0")
```

Comment:
the note in this PR about why v1.8.0 would be good to add here too, I think

Reply:
good point, comment added!